### DISCUSSION TITLE
标题应该提取独立存在的，不应该提取存在其他块中，例如 div 块中，则不应该被提取

### DISCUSSION THREAD

**Discussion by AndersonHJB** [vuepress/core#1201] [Q&A] [answered]
在其他内容「容器」里面的标题，不应该被右侧目录所提取。

也就是说：提取标题的时候应该只提取 nesting level为 0 的标题，嵌套在其他内容里的标题不应该被提取。

Titles in other content "containers" should not be extracted by the right directory.

That is, when extracting titles you should only extract titles that are at 0 level, and titles nested within other content should not be extracted.

![image](https://user-images.githubusercontent.com/48122016/207319217-d5db0788-ecbf-4e40-b5dc-ec39bf0d90df.png)

**Comment by meteorlxy** [comment, marked as answer]
That's a good point. Should be implemented in mdit-vue project。

--- update

Has been fixed by ecd3716e0fca2e4c4457d514f245ca06e6e9086e

**Comment by nibazshab** [comment]
这样的 Question 1，Question 2 容器是怎么搞出来的？没看到有关这种样式的容器的语法啊

How to do the “containers” like this ? I don't see the syntax for this type of container

**Reply by Mister-Hope** [reply]
See VuePress-plugin-md-enhance

**Reply by nibazshab** [reply]
OK, thank you, now i find it
